--- a/build.rs
+++ b/build.rs
@@ -7,6 +7,7 @@
 pub const DATA_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/data");
 
 fn main() -> Result<()> {
+    eprintln!("=== OUT_DIR: {:?}", std::env::var("OUT_DIR"));
     setup_po()?;
     setup_schemas("com.stremio.Stremio.gschema.xml")?;
 
@@ -26,7 +27,7 @@
             && extension == "po"
             && let Some(po_lang) = path.file_stem()
         {
-            let mo_dir = po_dir.join(po_lang).join("LC_MESSAGES");
+            let mo_dir = PathBuf::from(std::env::var("OUT_DIR").unwrap()).join(po_lang).join("LC_MESSAGES");
 
             fs::create_dir_all(&mo_dir)?;
 
@@ -46,7 +47,7 @@
 fn setup_schemas(filename: &str) -> Result<()> {
     println!("cargo:rerun-if-changed={DATA_DIR}");
 
-    let out_dir = dirs::data_dir().expect("Failed to get data dir");
+    let out_dir = PathBuf::from(std::env::var("OUT_DIR").unwrap());
     let out_dir = out_dir.join("glib-2.0/schemas");
 
     fs::create_dir_all(&out_dir)?;
